frontend/src/App.js

```diff
@@ -38,6 +42,7 @@ type InternalProps = {
 class App extends React.Component<InternalProps> {
     componentDidMount() {
         this.props.dispatch(locales.actions.get());
+        this.props.dispatch(project.actions.get(this.props.parameters.project));
```

Comment:
I'm curious if that's also where I should [load resources](https://github.com/mozilla/pontoon/pull/1292/files#diff-9d4d0fc9d1fef199d928b85a28631f6dR38)?

Reply:
Yeah, I think it makes sense to make this the place where we load all initial data. It will also make it easier to optimize in the future (we might want to make all these calls into a single GraphQL query, and split the results afterwards).